src/analytics.ts

```diff
@@ -0,0 +1,86 @@
+import posthog from 'posthog-js'
+import "vanilla-cookieconsent/dist/cookieconsent.css";
+import * as CookieConsent from "vanilla-cookieconsent";
+import { useEffect } from 'react';
+
+export async function enablePosthog() {
+  console.log("Enabling PostHog")
+  posthog.init(process.env.POSTHOG_API_KEY || 'phc_tRkyMnTTjz6EXTGDxTBw4FgZacvm3MdsXd7aK485Gnh', {
```

Comment:
I'd rather not use an env variable and also hard-code the key here explicitly. It seems a little redundant, and means we don't know if the env variable is even working? I have a hunch it might need a `REACT_APP_` prefix.

Reply:
Yeah it is redundant, this was just the quickest approach to getting the code out there.